### ISSUE TITLE
prevent the use of mentions for names

### ISSUE THREAD

**Issue by SheepTester** [Orbiit/giggle-bot#2]
@Velosify's name is `@everyone` which means he constantly spams everyone else with mentions

one lazy solution is to just blacklist "@"; I don't see any harm in doing that :P

**Comment by SheepTester** [comment]
to do so, one can add:
```js
if (~setName[1].indexOf("@")) {
  sendOK = false;
  msg.react(thumbs_down);
  channel.send(`i have been instructed to disallow the usage of @ in names, sorry!`);
} else {
  // ...
}
```
around [line 322](https://github.com/Orbiit/giggle-bot/blob/master/main.js#L322)

**Comment by SheepTester** [comment]
fixed by 5af714d

**Comment by heaucques** [comment]
aw
